### PR TITLE
PIN-3679 Add IPA code to contract PDF

### PR DESCRIPTION
**Pull request by nttdata-rtorsoli**

### CODE REVIEW

File: src/main/resources/agreementTemplate/index.html [hunk 1]
```diff
@@ -42,9 +42,9 @@ <h1>Richiesta di fruizione</h1>
     In data <strong>${todayDate}</strong> alle ore <strong>${todayTime}</strong>, l’Infrastruttura ha ricevuto la
     richiesta di fruizione contraddistinta da id <strong>${agreementId}</strong> (di seguito “Richiesta di Fruizione”),
     inviata dall’aderente
-    <strong>${consumerName}</strong> (di seguito “Fruitore”), tramite l’operatore amministrativo
+    <strong>${consumerName} (codice IPA: ${consumerValue}) </strong> (di seguito “Fruitore”), tramite l’operatore amministrativo
```
Comment (galales) [inline src/main/resources/agreementTemplate/index.html]:
```suggestion
    <strong>${consumerName} (codice IPA: ${consumerIPACode}) </strong> (di seguito “Fruitore”), tramite l’operatore amministrativo
```

File: src/main/resources/agreementTemplate/index.html [hunk 2]
```diff
@@ -42,9 +42,9 @@ <h1>Richiesta di fruizione</h1>
...
     <strong>${submitter}</strong>,
-    per l’E-service <strong>${eServiceName}</strong> reso disponibile dall’aderente <strong>${producerName}</strong>
+    per l’E-service <strong>${eServiceName}</strong> reso disponibile dall’aderente <strong>${producerName} (codice IPA: ${producerValue})</strong>
```
Comment (galales) [inline src/main/resources/agreementTemplate/index.html]:
```suggestion
    per l’E-service <strong>${eServiceName}</strong> reso disponibile dall’aderente <strong>${producerName} (codice IPA: ${producerIPACode})</strong>
```

File: src/main/scala/it/pagopa/interop/agreementprocess/service/util/PDFPayload.scala [hunk 3]
```diff
@@ -16,6 +16,8 @@ final case class PDFPayload(
   eService: String,
   producerName: String,
   consumerName: String,
+  producerValue: String,
+  consumerValue: String,
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/agreementprocess/service/util/PDFPayload.scala]:
```suggestion
  producerIPACode: String,
  consumerIPACode: String,
```

**Review by galales — COMMENTED** [review]
[same consideration](https://github.com/pagopa/interop-be-purpose-process/pull/172#pullrequestreview-1547692791) for private organizations

**Review by beetlecrunch — APPROVED** [review]
Approving, but I agree with @galales suggestions

**Review by TonioGela — APPROVED** [review]
Same 👆

**Review by galales — APPROVED** [review]

**Pull request merged**